xml/depl_nodes.xml

```diff
@@ -5002,6 +5008,461 @@ getent group postgres | cut -d: -f3</screen>
+     creating an elasticsearch snapshot. --> Must contain at least one entry.
+        </para>
+      </listitem>
+      </varlistentry>
+    </variablelist>
+
+    <bridgehead renderas="sect3"><guimenu>elasticsearch_curator: settings for
+    elastisearch-curator</guimenu></bridgehead>
+
+    <para>
+    <systemitem>elasticsearch-curator</systemitem> removes old and large
+    elasticsearch indices. The settings below  determine its behavior.
+    </para>
+
+    <variablelist>
+      <varlistentry>
+      <term>delete_after_days</term>
+      <listitem>
+        <para>
+	 Time threshold for deleting indices. Indices older the specified
+	 number of days are deleted. This parameter is unset by default, so indices are kept indefinitely.
+        </para>
+      </listitem>
+      </varlistentry>
+      <varlistentry>
+      <term>delete_after_size</term>
+      <listitem>
+        <para>
+	 Maximum size in megabytes of indices. Indices larger than the specified
+	 size are deleted. This parameter is unset by default, so indices are
+	 kept irrespective of their size.
+        </para>
+      </listitem>
+      </varlistentry>
+      <varlistentry>
+      <term>delete_exclude_index</term>
+      <listitem>
+        <para>
+	 List of indices to exclude from
+	 <systemitem>elasticsearch-curator</systemitem> runs. By default, only
+	 the <filename>.kibana</filename> files are excluded.
+        </para>
+      </listitem>
+      </varlistentry>
+      <varlistentry>
+      <term>cron_config</term>
+      <listitem>
+        <para>
+	 Specifies when to run
+	 <systemitem>elasticsearch-curator</systemitem>. Attributes of this
+	 parameter correspond to the fields in <systemitem>crontab(5)</systemitem>.
+        </para>
+      </listitem>
+      </varlistentry>
+    </variablelist>
+
+<bridgehead renderas="sect3"><guimenu>kafka: tunables for
+Kafka</guimenu></bridgehead>
+
+<variablelist>
+  <varlistentry>
+      <term>log_retention_hours</term>
+      <listitem>
+        <para>
+	 Number of hours for retaining log segments in Kafka's on-disk
+	 log. Messages older than the specified value are dropped.
+        </para>
+      </listitem>
+   </varlistentry>
+  <varlistentry>
+      <term>log_retention_bytes</term>
+      <listitem>
+        <para>
+	 Maximum size for Kafka's on-disk log in bytes. If the log grows beyond
+	 this size, the oldest log segments are dropped.
+        </para>
+      </listitem>
+   </varlistentry>
+ </variablelist>
+
+<bridgehead renderas="sect3"><guimenu>master: configuration for
+monasca-installer on the &crow; node</guimenu></bridgehead>
+
+<variablelist>
+  <varlistentry>
+      <term>influxdb_retention_policy</term>
+      <listitem>
+        <para>
+	 Number of days to keep metrics records in influxdb. Refer to <link
+	 xlink:href="https://docs.influxdata.com/influxdb/v0.9/query_language/database_management/#create-retention-policies-with-create-retention-policy"/>
```

Comment:
I'm not sure how stable that link is. Maybe it's better to copy the relevant parts to our documentation (we only need the time unit specification). The CMM operator's guide did that as well:

https://github.com/FujitsuEnablingSoftwareTechnologyGmbH/cmm-documentation/blob/master/DocSources/cmmoperator/install/c-cmmoinstcmm.xml#L118